changelogs/fragments/49-fix-lb-and-cert-info.yaml

```
@@ -0,0 +1,3 @@
+bugfixes:
+  - digital_ocean_load_balancer_info - Fix retrieving load balancer by ID (https://github.com/ansible-collections/community.digitalocean/issues/35).
+  - digital_ocean_certificate_info - Fix retrieving certificate by ID.
```

```suggestion
  - digital_ocean_load_balancer_info - fix retrieving load balancer by ID (https://github.com/ansible-collections/community.digitalocean/issues/35).
  - digital_ocean_certificate_info - fix retrieving certificate by ID (https://github.com/ansible-collections/community.digitalocean/issues/35).
```
(See https://docs.ansible.com/ansible/devel/community/development_process.html#changelog-fragment-entry-format)

---

Ah, thanks, I didn't know they should start lower case.